packages/utils/logger/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import * as winston from 'winston';
+
+import * as formats from './formats';
```

Comment:
You could re-export * as formats without importing it first

Reply:
I did the change but not a fan honeslty I think we loose in readable in this case

Reply:
Yes, I kinda agree, tbh I was just pointing out the fact that we could, not that we should :slightly_smiling_face: